unity-client/Assets/Scripts/MainScripts/DCL/Components/BuilderInWorld/DCLName.cs

```diff
@@ -44,30 +44,42 @@ public void SetNewName(string value)
         UpdateFromJSON(JsonUtility.ToJson(newModel));
     }
 
+    public void ForceSetNewName(string value)
+    {
+        Model newModel = new Model();
+        newModel.value = value;
+
+        ApplyNewValue(JsonUtility.ToJson(newModel));
+    }
+
     public override IEnumerator ApplyChanges(string newJson)
+    {
+        ApplyNewValue(newJson);
+        return null;
+    }
+
+    public void ApplyNewValue(string newJson)
```

Comment:
Changed `ApplyNewValue `to be private since it makes more sense.

However, there is a pending task of refactoring all the components to have a `SetModel` function so we can work with that function in all components.

Reply:
Sounds good!